src/index.ts

```diff
@@ -6,7 +6,7 @@ import Aws from 'serverless/plugins/aws/provider/awsProvider';
 import Service from 'serverless/classes/Service';
 
 type ConventionsConfig = {
-  ignore?: {
```

Comment:
In the serverless.yml we don't know if `ignore` will be defined or not so this seems like the use-case for an optional value. I know that TS will throw an error but is it better to be handling this error instead?

Reply:
@TheOrangePuff Please check my comment at https://github.com/aligent/serverless-conventions/pull/110#discussion_r991687944.
If `ignore` is not defined in `serverless.yml` then `conventionsConfig` will be `null`. It's a bit confuse because `ConventionsConfig` is the type of `conventionsConfig`, not the `this.serverless.service.initialServerlessConfig.conventions` which is the real object parsed out from `serverless.yml`